modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -572,13 +572,13 @@ export default class I3SConverter {
 
     let boundingVolumes = createBoundingVolumes(sourceTile, this.geoidHeightModel!);
 
-    const batchTable = sourceTile?.content?.batchTableJson;
+    const propertyTable = getPropertyTable(sourceTile);
```

Comment:
I don't see a reason of taking `propertyTable` to `i3s-converter` side. You are sending it back to `geometry-converter` later. I suggest make this variable internal in `geometry-converter`

Reply:
Actually I need propertyTable in i3s-converter file to generate node attributes and fill in some fields like `attributeStorageInfo`, `popupInfo` etc.
https://github.com/visgl/loaders.gl/blob/master/modules/tile-converter/src/i3s-converter/i3s-converter.ts#L1193